archiver.go

```diff
@@ -18,6 +17,9 @@ import (
 	"gopkg.in/src-d/go-git.v4/plumbing"
 	"gopkg.in/src-d/go-git.v4/plumbing/transport"
 	"gopkg.in/src-d/go-kallax.v1"
+
+	"github.com/inconshreveable/log15"
+	"github.com/jpillora/backoff"
```

Comment:
these should go before the gopkg.in imports

Reply:
Aren't we separating our libraries from third libraries in the imports declaration? :thinking:

Reply:
yes, but it's alphabetical and `github.com` < `gopkg.in`

Reply:
Ok, I'm changing it :+1: , anyway I thought that this convention was:
```go
import (
        "a-stdlib"
        "b-stdlib"
        "c-stdlib"

        "a-ourlib"
        "b-ourlib"
        "c-ourlib"

        "a-thirdpartylib"
        "b-thirdpartylib"
        "b-thirdpartylib"
)
```